src/Database/DriverInterface.php

```diff
@@ -25,9 +25,35 @@
  *
  * @method int|null getMaxAliasLength() Returns the maximum alias length allowed.
  * @method int getConnectRetries() Returns the number of connection retry attempts made.
+ * @method bool supports(string $feature) Checks whether a feature is supported by the driver.
  */
 interface DriverInterface
 {
+    /**
+     * @var string
+     */
+    public const FEATURE_QUOTING = 'quoting';
```

Comment:
Wouldn't all drivers support quoting? Or is this more of a 'is quoting enabled?' thing?

Reply:
That's from this existing function:
https://github.com/cakephp/cakephp/blob/4.next/src/Database/Driver.php#L304

Reply:
I totally forgot about that.